### PR TITLE
Contributing DendroPy

### PR DESCRIPTION
**Pull request by tloizou**
Adding DendroPy (Python Package) in EasyBuild

### CODE REVIEW

File: easybuild/easyconfigs/d/DendroPy/DendroPy-3.12.0-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+##
+
+easyblock = "PythonPackage"
+
+name = 'DendroPy'
+version = '3.12.0'
+
+homepage = 'https://pypi.python.org/pypi/DendroPy/'
+description = """A Python library for phylogenetics and phylogenetic computing: reading, writing, simulation, processing and manipulation of phylogenetic trees (phylogenies) and characters.
```
Comment (boegel) [inline easybuild/easyconfigs/d/DendroPy/DendroPy-3.12.0-goolf-1.4.10-Python-2.7.3.eb]: split lone line across multiple lines

File: easybuild/easyconfigs/d/DendroPy/DendroPy-3.12.0-goolf-1.4.10-Python-2.7.3.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+"""
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://pypi.python.org/packages/source/D/DendroPy/']
+sources = ['%s-%s.tar.gz' % (name, version)]
```
Comment (boegel) [inline easybuild/easyconfigs/d/DendroPy/DendroPy-3.12.0-goolf-1.4.10-Python-2.7.3.eb]: `SOURCELOWER_TAR_GZ`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a9ff6f3da03a1222c84e for a full test report.

**Comment by tloizou** [comment]
@boegel: changed some details in easyconfig. Ready for testing.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d2204352c96252aa3de9 for a full test report.

**Comment by boegel** [comment]
Looking good, thanks @tloizou!

**Pull request merged**